src/@next/components/organisms/StripePaymentGateway/types.ts

```diff
@@ -21,6 +23,16 @@ export interface IProps {
    * Method called after the form is submitted. Passed token attribute will be used to create payment.
    */
   processPayment: (token: string, cardData: ICardData) => void;
+  /**
+   * Method to call on gateway payment submission.
+   */
+  submitPayment: (data?: object) => Promise<any>;
```

Comment:
Same as earlier, would it be possible to be more specific about the Promise return type?

Reply:
fixed